tests/feature_config.rs

```diff
@@ -30,6 +40,14 @@ impl CargoSemverChecks {
 
 #[test]
 fn simple_only_explicit_feature() {
+    CargoSemverChecks::new_with_check_release_subcommand(
+        "test_crates/features_simple/new/",
+        "test_crates/features_simple/old/Cargo.toml",
+    )
+    .add_arg("--only-explicit-features")
+    .run()
+    .success();
```

Comment:
This duplication is a bit unfortunate: over time as we edit these tests and add new ones, we might update one version of the check but fail to update the other. This is a maintainability risk.

Could you think of another way to set up the test where we don't need to duplicate all the test code like this? Perhaps by doing some more refactoring of the `CargoSemverChecks` type?